app/services/moderator/merge_user.rb

```diff
@@ -67,7 +69,10 @@ def remove_additional_email
     end
 
     def merge_profile
-      @delete_user.github_repos&.update_all(user_id: @keep_user.id) if @delete_user.github_repos.any?
+      if @delete_user.github_repos.any?
+        @delete_user.github_repos&.update_all(user_id: @keep_user.id)
```

Comment:
I think `&.` is useless here since you check that `github_repos` exist before. And if you haven't `github_repos` would still return `ActiveRecord_Associations_CollectionProxy` which responds to `update_all`, and the check wouldn't be needed. 
(I see that it's not the change of this pr, but still kept the comment)